storm-client/src/jvm/org/apache/storm/serialization/KryoTupleDeserializer.java

```diff
@@ -39,7 +38,7 @@ public KryoTupleDeserializer(final Map<String, Object> conf, final GeneralTopolo
         _kryoInput = new Input(1);
     }        
 
-    public Tuple deserialize(byte[] ser) {
+    public TupleImpl deserialize(byte[] ser) {
```

Comment:
Thanks! Good to know. Do you know which version of JDK make it possible? Was it possible from very first version of Java?